lua/orgmode/org/autocompletion/omni.lua

```diff
@@ -112,7 +118,7 @@ local function omni(findstart, base)
     then
       local items = {}
       if context.fetcher then
-        items = context.fetcher(fetcher_ctx)
+        items = context.fetcher(url)
       else
         items = { unpack(context.list) }
       end
```

Comment:
The following lines expose a bug which I currently don't understand completely:
When I use file urls, which don't start with the `file:` prefix, omnicompletion only finds headlines when typing `::*` after the file. If I type `::#` I would expect omnicompletion to find custom_ids. They get removed here, because `base` is `:#` while it is just `*` when I aim for headlines (which matches the pattern in the filter).

I haven't understand, how the to cases differ regarding the calculation of `base`. @kristijanhusak Any ideas?

Reply:
Both cases work perfectly fine, if I have no file (the link starts with `#` or `*` respectively) or if the `file:` prefix is used.